omnibus/files/private-chef-scripts/gather-logs

```diff
@@ -55,12 +75,15 @@ tmpdir="$(mktemp -d)/$hostname/$timestamp"
 #RHEL5 has ip and other tools here
 PATH=$PATH:/bin:/sbin
 
+PATH=/opt/$path/embedded/bin:$PATH
+
 mkdir -p "$tmpdir"
 
-for i in /opt/$path*/version-manifest.txt \
+for i in /opt/{chef,$path}*/version-manifest.txt \
     /opt/$path/pc-version.txt \
     /etc/$path/$config_name \
-    /etc/$path/chef-server-running.json \
+    /etc/${path}*/*-running.json \
+    /var/opt/$path/rabbitmq/log/rabbit@localhost.log \
```

Comment:
Wasn't there a relatively recent PR merged to change the name of the rabbit node? Would this also cause the log file name to change?